__tests__/business-text/business-text.query.test.js

```diff
@@ -0,0 +1,304 @@
+/* eslint-disable no-undef */
+const { gql } = require('apollo-boost');
+const client = require('../../utils/apollo-test-client');
+require('dotenv').config();
+const { BUSINESS_TEXT_NOT_FOUND } = require('../../error-messages/business-text.messages');
+const { newBusinessText, notExistBusinessTextId } = require('./business-text.variables');
+
+let businessText = null;
+let businessTextId = '';
+
+describe('Contacts queries', () => {
+  beforeAll(async () => {
+    const res = await client
+      .mutate({
+        mutation: gql`
+          mutation($businessText: BusinessTextInput!) {
+              addBusinessText(businessText: $businessText) {
+                  ... on BusinessText {
+                      _id
+                      code
+                      title {
+                          value
+                          lang
+                      }
+                      text {
+                          value
+                          lang
+                      }
+                  }
+                  ... on Error {
+                      message
+                      statusCode
+                  }
+              }
+          }
+      `,
+        variables: {
+          businessText: newBusinessText,
+        },
+      })
+      .then(response => response)
+      .catch(e => e);
+
+    businessTextId = res.data.addBusinessText._id;
+  });
+
+  afterAll(async () => {
+    await client.mutate({
+      mutation: gql`
+          mutation($id: ID!) {
+              deleteBusinessText(id: $id) {
+                  ... on BusinessText {
+                      _id
+                      code
+                      title {
+                          value
+                          lang
+                      }
+                      text {
+                          value
+                          lang
+                      }
+                  }
+                  ... on Error {
+                      message
+                      statusCode
+                  }
+              }
+          }
+      `,
+      variables: { id: businessTextId },
+    });
+  });
+
+  test('#1 Should receive all business texts', async () => {
+    const res = await client
+      .query({
+        query: gql`
+          query {
+              getAllBusinessTexts {
+                  code
+                  title {
+                      value
+                      lang
+                  }
+                  text {
+                      lang
+                      value
+                  }
+              }
+          }
+      `,
+      })
+      .catch(e => e);
+
+    const allTexts = res.data.getAllBusinessTexts;
+    const justAddedText = allTexts[allTexts.length - 1];
+
+    expect(allTexts).toBeDefined();
+    expect(justAddedText).toEqual({
+      __typename: 'BusinessText',
+      title: [
+        {
+          __typename: 'Language',
+          value: 'НоваБТ',
```

Comment:
Please, use here values from the `newBusinessText` instead of plain hardcode literals. It would be much easier and safe to changes some values in `newBusinessText` and be sure that it is changes in tests too. Use hardcode values only for properties that you cannot take from `newBusinessText`